public/components/overview/components/card_configs.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { i18n } from '@osd/i18n';
+
+// Plugin URLs
+export const gettingStartedURL = 'observability-gettingStarted';
+const discoverURL = 'data-explorer/discover';
+const metricsURL = 'observability-metrics';
+const tracesURL = 'observability-traces-nav#/traces';
+const servicesURL = 'observability-services-nav#/services';
+const alertsURL = 'alerting';
+const anomalyDetectionURL = 'anomaly-detection-dashboards';
```

Comment:
Should these be moved to common constants? some of them might already exist there.

Reply:
Moved to constants. No matching pre-existing urls.

Reply:
Moved / re-used variables from shared constants.